exe-common/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 extern crate cardano;
 extern crate cardano_storage;
+#[macro_use]
```

Comment:
Required for cbor! macro which we need in various places to get raw buffers to sign out of components of blocks.